subworkflows/nf-core/utils_nextflow_pipeline/main.nf

```diff
@@ -15,10 +15,10 @@ import nextflow.extension.FilesEx
 workflow UTILS_NEXTFLOW_PIPELINE {
 
     take:
-    print_version           // boolean: print version
-    dump_parameters         // boolean: dump parameters
-    outdir                  // path: base directory used to publish pipeline results
-    check_conda_channels    // boolean: check conda channels
+    print_version        // boolean: print version
+    dump_parameters      // boolean: dump parameters
+    outdir               //    path: base directory used to publish pipeline results
```

Comment:
that's a new rule of the [Harshil Alignment™️](https://nf-co.re/docs/contributing/modules#what-is-the-harshil-alignment), no?

Reply:
cc @jfy133

Reply:
*sigh* you made me do it... https://github.com/nf-core/website/pull/2205